src/index.html

```diff
@@ -22,6 +22,381 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="/src"
+      >
+        <img
+          src="images/logo.png"
+          alt="moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="/apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="/samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```